balancer/rls/internal/keys/builder.go

```diff
@@ -27,6 +27,8 @@ import (
 	"sort"
 	"strings"
 
+	"github.com/google/go-cmp/cmp"
+
```

Comment:
Empty line.

(File watcher is indeed not working :))

Reply:
Hmm ... I think I might have introduced in the empty line in the first place because I copy pasted stuff from another branch. But once the empty line is present, it does not delete it. So, I had to manually delete this import line and then hit "Save" and now the empty line is gone.